subsys/secure_storage/Kconfig.its_store

```diff
@@ -60,8 +60,23 @@ endif # SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_ZMS
 
 if SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS
 
+config SECURE_STORAGE_ITS_STORE_SETTINGS_NAME_CUSTOM
+	bool "Custom naming scheme for the stored settings"
+	help
+	  Implement the secure_storage_its_store_settings_get_name() function
+	  declared in <zephyr/secure_storage/its/store/settings_get.h>
```

Comment:
I am not a big fan of putting function /file names / paths. This can change and gets inconsistent. Besides that this should describe the functionality. Just say for what this symbol is used.